lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -56,13 +56,22 @@
   @include nx-text-helpers.truncate-ellipsis;
   @include nx-text-helpers.regular;
 
-  border: none;
+  border: 1px solid transparent;
   cursor: pointer;
   display: block;
   font-size: var(--nx-font-size);
   line-height: var(--nx-line-height);
   padding: var(--nx-spacing-base) var(--nx-spacing-4x);
   text-decoration: none;
+
+  &:focus {
+    border: 1px solid var(--nx-color-interactive-border-focus);
+    box-shadow: var(--nx-box-shadow-focus);
+
+    &, &:hover, &:active {
+      background-color: transparent;
```

Comment:
I'm not sure this is desirable.  The mockups tend not to have combinations of states like focus+hover or focus+clicked, but I would assume we want both of those states to do what they usually do.  It seems odd to have an element not visually respond to hover or click just because it is already focused.  @alexalyons126 what do you think?

Reply:
> I'm not sure this is desirable. The mockups tend not to have combinations of states like focus+hover or focus+clicked, but I would assume we want both of those states to do what they usually do. It seems odd to have an element not visually respond to hover or click just because it is already focused. @alexalyons126 what do you think?

As discussed and confirmed by @alexalyons126, the default hover and active background colors should be added back when in focus state.
Fixed in this commit:
303c2c53ee001bc84189c39268c7a2621f2f26e4